File: simple.js
var pg = require('pg');
var async = require('async');

pg.connect(function(err, client, done) {
  if(err) throw err;
  var getAll = function(n, cb) {
    client.query('SELECT * FROM test_data', cb);
  };
  async.timesSeries(20, getAll, function(err) {
    if(err) throw err;
    var start = Date.now();
    client.query('SELECT * FROM test_data', function() {
      console.log('node',Date.now() - start, 'milliseconds');
      done();
      pg.end();
    });
  });
});

File: simple.rb
require 'pg'
pg = PGconn.open()

20.times do
  res = pg.exec "SELECT * FROM test_data"
  res.count
end

start = Time.now
res = pg.exec "SELECT * FROM test_data"
res.count
time = Time.now - start
puts "ruby #{time * 1000} milliseconds"
